packages/webpack-pwa-manifest-plugin/tests/runTest.js

```diff
@@ -0,0 +1,67 @@
+const webpack = require('webpack');
+const path = require('path');
+const ls = require('list-directory-contents');
+const fs = require('fs');
+const assert = require('assert');
+const chalk = require('chalk');
+
+function run(name, next) {
```

Comment:
seems like it would be easier to make this an async function, also that way it could return a value if needed. (would need to util.promisify(webpack) probably)